### PR TITLE
refactor: widen PyYAML pin to >=5.0,<7.0 [APE-947]

### PR DESCRIPTION
**Pull request by fubuloubu**
### What I did

`awscli` depends on an older version of pyyaml, and I don't think we rely on any advanced behavior of this dependency, so opening up the pin seems safe

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -104,7 +104,7 @@
         "PyGithub>=1.54,<2",
         "pytest>=6.0,<8.0",
         "python-dateutil>=2.8.2,<3",
-        "pyyaml>=6.0,<7",
+        "PyYAML>=5.0,<7",
```
Comment (antazoey) [inline setup.py:107]: i never realized these were case insensitive
some of the others still using casing tho

should we make all of them lower case or should we stick to the rule of using actual pypi names?
kinda nice to know what the actual pypi name is (with casing)

**Review by antazoey — APPROVED** [review]
is good but see comment about casing

**Comment by antazoey** [comment]
@fubuloubu merge this one too?

File: setup.py (hunk 1 above)
Comment (fubuloubu) [reply setup.py:107]: Would use the pypi name, but can update later no biggie

**Comment by fubuloubu** [comment]
> @fubuloubu merge this one too?

Yes I think this one is good to go, would help me with a pinning issue in chaos

File: setup.py (hunk 1 above)
Comment (antazoey) [reply setup.py:107]: oh i see, no worries then

**Pull request merged**